pinc/charsuite-combined.inc

```diff
@@ -0,0 +1,32 @@
+<?php
+include_once($relPath."CharSuites.inc");
+
+$charsuite = new CharSuite("combined", _("Combined"));
+$charsuite->codepoints= [
+    'U+004E',
```

Comment:
We could have some way to check if these are normalised forms. (The UI I made earlier revealed that  4e>307 is 1e44 and 4e>30c is 147).

Reply:
Indeed - I stumbled over this just a moment ago. I added normalization into `utf8_combined_chr()` in the common functions MR.